README.md

```diff
@@ -14,6 +14,8 @@ while coLaboratory Classic executes code via local Jupyter kernels
 (such as the IPython kernel) that have complete access to the host
 system and files.
 
+*CoLaboratory is no longer being actively developed or maintained.  We will consider minor pull requests, but most of the CoLaboratory team's focus is now on integration of Google Drive with IPython/Jupyter.  For more information on this project see https://github.com/jupyter/jupyter-drive*
```

Comment:
I'd link jupyter-drive directly in the second sentence and delete the last sentence:

```markdown
We will consider minor pull requests, but most of the CoLaboratory team's focus is
now on [integration of Google Drive with IPython/Jupyter](https://github.com/jupyter/jupyter-drive).
```

If you want to add some pizzazz, wrap it all in :warning: :warning: :warning:.